readme.md

```diff
@@ -274,13 +274,13 @@ This importer will extract files from all images it can find in the `install.wim
 This importer utilizes 7z to recursively extract contents of Windows Update packages. It will look for Windows Update files in the provided GCS bucket, easiest way to automatically update the GCS bucket with new updates would be to do the following: 
 
 1. Set up GCE VM running Windows Server in hashr GCP project. 
-1. Configure it with WSUS role, select Windows Update packages that you'd like to process
-1. Configure WSUS to automatically approve and download updates to local storage 
-1. Set up a Windows task to automatically sync content of the local storage to the GCS bucket: `gsutil -m rsync -r D:/WSUS/WsusContent gs://hashr-wsus/` (remember to adjust the paths)
+2. Configure it with WSUS role, select Windows Update packages that you'd like to process
+3. Configure WSUS to automatically approve and download updates to local storage 
+4. Set up a Windows task to automatically sync content of the local storage to the GCS bucket: `gsutil -m rsync -r D:/WSUS/WsusContent gs://hashr-wsus/` (remember to adjust the paths)
 
 ### Setting up Postgres exporter 
 
-Postgres exporter allows to send hashes, file metadata and the actual content of the file to a PostgreSQL instance. For best performance it's advised to set it up on a separate and dedicated machine. 
+Postgres exporter allows sending of hashes, file metadata, and the actual content of the file to a PostgreSQL instance. For best performance it's advised to set it up on a separate and dedicated machine. 
```

Comment:
I don't think we put comma before "and" in English in this context, ref https://www.grammarly.com/blog/comma-before-and/

Reply:
All right. I have removed the comma. Thank you.